examples/tools/openapi.ts

```diff
@@ -0,0 +1,36 @@
+import "dotenv/config.js";
+import { BeeAgent } from "bee-agent-framework/agents/bee/agent";
+import { TokenMemory } from "bee-agent-framework/memory/tokenMemory";
+import { OllamaChatLLM } from "bee-agent-framework/adapters/ollama/chat";
+import { OpenAPITool } from "bee-agent-framework/tools/openapi";
+import * as fs from "fs";
+
+async function fetchSchema() {
+  try {
+    const fileContent = fs.readFileSync("examples/tools/github_schema.json", "utf-8");
+    return fileContent;
+  } catch (error) {
+    console.error("An error occurred:", error);
+  }
+}
+const spec = await fetchSchema();
```

Comment:
- Use `fs.promises.readFile` with `await`; you do not need any function.
- use relative path